{
  "uuid": "6a101936-94ce-4af2-8c0f-6e19b8acbfa2",
  "path": "/lib/python3/dist-packages/gi/__pycache__/_error.cpython-312.pyc",
  "size": 1993,
  "modified": "2025-02-15T08:09:33Z",
  "created": "2025-05-09T05:07:26.889790731Z",
  "sha256_hash": "e471a64fabb8ee8bd9910acb0b358f7a1e374503ac91da7d4301a5d11ba85780",
  "file_type": {
    "Other": "pyc"
  },
  "permissions": 33188,
  "is_executable": false,
  "is_signed": false,
  "code_signature": null,
  "bundle_info": null,
  "scan_timestamp": "2025-06-01T18:46:33.951152414Z"
}